Android/app/src/main/java/fashiome/android/activities/ProductFormActivity.java

```diff
@@ -380,7 +380,7 @@ public void onConnected(Bundle dataBundle) {
         if (location != null) {
             Toast.makeText(this, "GPS location was found! Lat:"+ location.getLatitude() +" Long:"+location.getLongitude(), Toast.LENGTH_SHORT).show();
             LatLng latLng = new LatLng(location.getLatitude(), location.getLongitude());
-            product.setAddress(new Address(latLng));
+            product.setAddress(new Address(Item.getRandomLocation(latLng.latitude, latLng.longitude,5000)));
```

Comment:
Is this adding a random location still ? Product form activity is working fine with users current location why we need random location ?